SECURITY_POLICY.md

```diff
@@ -0,0 +1,62 @@
+# Security
+
+The Galaxy project is strongly committed to security and responsible disclosure. We have adopted and published a set of policies specifying how we will act in response to reported security issues, in order to ensure timely updates are made available to all affected parties.
+
+## Reporting Security Issues
+
+If you believe you have discovered a security issue, please email [galaxy-committers@lists.galaxyproject.org](galaxy-committers@lists.galaxyproject.org). We ask that you not disclose the issues on the public issue tracker. Someone on that list will acknowledge your email within 2 US business days.
+
+Security issues which affect a pre-release version of Galaxy (i.e. the dev branch in GitHub) do not need to go through this process, you may open issues and PRs publicly.
+
+## Supported versions
+
+The following branches or releases receive security support:
+
+- Development on the `dev` branch, hosted on GitHub, which will become the next release of Galaxy
+- Releases within the past 12 months.
+  - E.g. 16.04 will receive support until 2017-04. As the month changes to 2017-05 it will become unsupported.
+- There are currently no plans for LTS releases.
+
+For unsupported branches:
+
+- Older versions of Galaxy may be affected by security issues.
+- Security patches *may* apply
+- The security team will not investigate issues that pertain to unsupported releases.
```

Comment:
maybe change the language to something like `does not commit to investigate` instead of `will not`?